main.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
you said the code isnt for the project right? you can open python file in scratches and it wont upload to git :)